tests/data/images/generate_assets.py

```
@@ -0,0 +1,108 @@
+
+
+def font(size):
+    try:
+        return ImageFont.truetype(FONT_PATH, size)
+    except OSError:
+        return ImageFont.load_default()
+
+
+def centered(draw, box, text, fnt, fill):
+    x0, y0, x1, y1 = box
+    l, t, r, b = draw.textbbox((0, 0), text, font=fnt)
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)